tests/foreman/cli/test_ping.py

```diff
@@ -35,13 +33,13 @@ def test_positive_ping():
 
     :expectedresults: hammer ping returns a right return code
     """
-    result = ssh.command('hammer ping')
-    assert result.stderr.decode() == ''
+    result = default_sat.execute('hammer ping')
+    assert result.stderr[1].decode() == ''
```

Comment:
again the index, looks weird, please add comments with how the structure of the stdout, stderrr, ... looks like